shared/actions/wallets.tsx

```diff
@@ -1502,28 +1502,34 @@ const handleSEP6Error = (err: RPCError) => [
 ]
 
 const assetDeposit = (_: TypedState, action: WalletsGen.AssetDepositPayload) =>
-  RPCStellarTypes.localAssetDepositLocalRpcPromise({
-    accountID: action.payload.accountID,
-    asset: assetDescriptionOrNativeToRpcAsset(
-      Constants.makeAssetDescription({
-        code: action.payload.code,
-        issuerAccountID: action.payload.issuerAccountID,
-      })
-    ),
-  })
+  RPCStellarTypes.localAssetDepositLocalRpcPromise(
+    {
+      accountID: action.payload.accountID,
+      asset: assetDescriptionOrNativeToRpcAsset(
+        Constants.makeAssetDescription({
+          code: action.payload.code,
+          issuerAccountID: action.payload.issuerAccountID,
+        })
+      ),
+    },
+    Constants.assetDepositWaitingKey(action.payload.issuerAccountID)
```

Comment:
is this key supposed to be unique?  if so, we should add the action.payload.code as well.

Reply:
@patrickxb Thanks, good catch, made this change!

Yes, it's supposed to uniquely identify one visible button on the screen to make it spin. So, I think if we'd had two assets, both expanded and showing the buttons, both with the same issuerAccountID but different asset codes, then clicking Deposit on one would have caused both to spin at the same time.